subprojects/launcher/src/integTest/groovy/org/gradle/launcher/continuous/SmokeContinuousIntegrationTest.groovy

```diff
@@ -82,8 +90,10 @@ class SmokeContinuousIntegrationTest extends AbstractContinuousIntegrationTest {
         waitBeforeModification(markerFile)
         markerFile.text = "changed"
         then:
-        buildTriggeredAndSucceeded()
-        output.contains "value: changed"
+        noBuildTriggered()
+        // TODO: We may want to support this use-case at some point
+        //   buildTriggeredAndSucceeded()
+        //   output.contains "value: changed"
```

Comment:
Let's add some description here why this doesn't work.

Reply:
Done.